### DISCUSSION TITLE
Why there is a line and sentence on my each post?

### DISCUSSION THREAD

**Discussion by M4TRIX04** [nunocoracao/blowfish#1890] [General]
Hi, you guys,
Could you tell me Why there is a line and sentence on my each post?  How to remove it or disable it.

![Untitled-1](https://github.com/user-attachments/assets/4c0f89fb-1a63-4f28-a27a-7f7f6ba892d3)

**Comment by nozomizore1974** [comment]
This line and the words below mean you are using a _list page_ rather than an *article page*.
So I guess your post file be named as "_index.md"?

**Reply by M4TRIX04** [reply]
Oh, Thank you, sir.  Yes, my file name was _index.md. So if I create a single page like the about page,  I should name the file "index.md".

**Reply by nozomizore1974** [reply]
Exactly you can use any name you like for an article page except *_index.md*.
Have a nice blowfish~
:-)
